### PR TITLE
Feature - Webpack build changes to support css modules

### PR DESCRIPTION
**Pull request by Tomdye**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

- Adds ability to generate css-module output for projects using the CLI.
- IDE changed spaces to tabs

Resolves: https://github.com/dojo/widgets/issues/224
Supersedes: https://github.com/dojo/cli-build/pull/20 

Note: Typed CSS modules are outside the scope of this initial delivery

### CODE REVIEW

File: src/webpack.config.js [hunk 1]
```diff
@@ -48,76 +56,94 @@ module.exports = function (args) {
...
+			libraryTarget: 'umd',
+			path: path.resolve('./dist'),
+			filename: '[name].js'
+		},
+		devtool: 'source-map',
+		resolve: {
+			root: [ basePath ],
+			extensions: ['', '.ts', '.tsx', '.js', '.css.json'],
+			alias: {
+				rxjs: '@reactivex/rxjs/dist/amd'
```
Comment (dylans) [inline src/webpack.config.js]: Do we need this any more?

File: src/webpack.config.js [hunk 2]
```diff
@@ -48,76 +56,94 @@ module.exports = function (args) {
...
+		],
+		output: {
+			libraryTarget: 'umd',
+			path: path.resolve('./dist'),
+			filename: '[name].js'
+		},
+		devtool: 'source-map',
+		resolve: {
+			root: [ basePath ],
+			extensions: ['', '.ts', '.tsx', '.js', '.css.json'],
```
Comment (dylans) [inline src/webpack.config.js]: Probably don't need tsx, but maybe better to include it?

File: src/webpack.config.js (hunk 1 above)
Comment (Tomdye) [inline src/webpack.config.js]: Not sure, but I don't think removing it would be in the scope of this PR

File: src/webpack.config.js (hunk 2 above)
Comment (Tomdye) [inline src/webpack.config.js]: not sure why it's there, but again probably not a change that's in the scope of this PR

**Pull request merged**